### PR TITLE
Soften language to discourage AAD Connect usage

### PR DESCRIPTION
**Pull request by wrall**
Addresses #2089

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by al3teoi11** [comment]
احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: William Rall <notifications@github.com>
‏‏تم الإرسال: Friday, July 31, 2020 12:16:58 AM
إلى: MicrosoftDocs/OfficeDocs-Exchange <OfficeDocs-Exchange@noreply.github.com>
نسخة: Subscribed <subscribed@noreply.github.com>
‏‏الموضوع: [MicrosoftDocs/OfficeDocs-Exchange] Soften language to discourage AAD Connect usage (#2161)


Addresses #2089<https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2089>

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/MicrosoftDocs/OfficeDocs-Exchange/pull/2161

Commit Summary

  *   Soften language to discourage AAD Connect usage

File Changes

  *   M Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md<https://github.com/MicrosoftDocs/OfficeDocs-Exchange/pull/2161/files#diff-d0e80bca213eb4e2d0e03ac7e6012ddb> (4)

Patch Links:

  *   https://github.com/MicrosoftDocs/OfficeDocs-Exchange/pull/2161.patch
  *   https://github.com/MicrosoftDocs/OfficeDocs-Exchange/pull/2161.diff

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ Once your G Suite environment has been properly configured, you can complete you
...
... you may be able to deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users.  See [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for setup instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enable your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md). Once the Mail Users have been created in Microsoft 365, the Azure AD Connect may need to be disabled in order to allow the migration process to convert these users into mailboxes - see [Turn off directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/turn-off-directory-synchronization).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:197]:
```suggestion
For more advanced scenarios, you may be able to deploy Azure Active Directory (Azure AD) Connect to provision your Mail Users. See [Deploy Microsoft 365 Directory Synchronization in Microsoft Azure](https://docs.microsoft.com/office365/enterprise/deploy-office-365-directory-synchronization-dirsync-in-microsoft-azure) for an overview, and [Set up directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/set-up-directory-synchronization) for setup instructions. Then, you need to deploy an Exchange server in your on-premises environment for user management, and mail-enable your users using this server. For more information, see [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/exchange/decommission-on-premises-exchange) and [Manage mail users](/Exchange/ExchangeServer/recipients/mail-users.md). Once the Mail Users have been created in Microsoft 365, the Azure AD Connect may need to be disabled in order to allow the migration process to convert these users into mailboxes - see [Turn off directory synchronization for Microsoft 365](https://docs.microsoft.com/office365/enterprise/turn-off-directory-synchronization).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:197]: Please remove extra space at the end of the first sentence.

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**